components/gif_picker/components/InfiniteScroll/index.ts

```diff
@@ -78,7 +79,7 @@ export default class InfiniteScroll extends PureComponent {
     setScrollPosition() {
         const {scrollPosition} = this.props;
         if (scrollPosition !== null) {
-            window.scrollTo(0, scrollPosition);
+            window.scrollTo(0, scrollPosition!);
```

Comment:
(nitpick) this here happens because the type of `scrollPosition` is `number | undefined`. If the condition is changed to `if (scrollPosition)` to cover null and undefined, there's not need for the exclamation marl

Reply:
Updated 💯